pkg/cmd/registry/artifact/crud/create/create.go

```diff
@@ -177,5 +181,34 @@ func runCreate(opts *options) error {
 	}
 	opts.Logger.Info(opts.localizer.MustLocalize("artifact.common.message.created"))
 
+	err = printBrowserUrl(opts, &metadata)
+	if err != nil {
+		return err
+	}
+
 	return dump.Formatted(opts.IO.Out, opts.outputFormat, metadata)
 }
+
+func printBrowserUrl(opts *options, metadata *registryinstanceclient.ArtifactMetaData) error {
+	conn, err := opts.Connection(connection.DefaultConfigRequireMasAuth)
+	if err != nil {
+		return err
+	}
+
+	registry, _, err := serviceregistry.GetServiceRegistryByID(opts.Context, conn.API().ServiceRegistryMgmt(), opts.registryID)
+	if err != nil {
+		return err
+	}
+
+	group := metadata.GetGroupId()
+
+	if group == "" {
+		group = "default"
```

Comment:
We should probably convert that to a constant.

Reply:
There is one :D

Reply:
Yes but it is not being used here.